index.md

```diff
@@ -54,7 +54,7 @@ sections:
       - title: "Language Spec"
         description: "Scala's formal language specification."
         icon: "fa fa-book"
-        link: http://scala-lang.org/files/archive/spec/2.12/
+        link: https://scala-lang.org/files/archive/spec/2.12/
```

Comment:
Could you change the link to the spec for scala 2.13: `https://scala-lang.org/files/archive/spec/2.13/` ?